plugins/bean-validation/src/test/java/org/apache/struts/beanvalidation/models/Address.java

```diff
@@ -20,11 +20,12 @@
 
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
+import javax.validation.groups.Default;
 
 public class Address {
 
-    @NotNull(message = "streetNotNull")
-    @Size(min = 3, max = 64, message = "streetSize")
+    @NotNull(message = "streetNotNull", groups = {Default.class, Person.StreetChecks.class, Person.NameAndStreetChecks.class})
+    @Size(min = 3, max = 64, message = "streetSize", groups = {Default.class, Person.StreetChecks.class, Person.NameAndStreetChecks.class})
```

Comment:
`Default.class` is required?

Reply:
Yes, 
see Examples http://beanvalidation.org/1.1/spec/#validationapi-validatorapi-groups
... groups = {Default.class, Person.StreetChecks.class, Person.NameAndStreetChecks.class}) means, this Constraints is valid for on of them.

If validator called without a group ...validator.validate(foo);... it validates the group Default (implicitly). 

Testcase 'testModelDrivenActionSize' would fail

Reply:
Ok, I think I understand, `Default.class` is used when no group was defined but I would not force users to put `Default.class` in the existing annotations - this can break backward comaptibility. Instead it should be defined inside the annotation.

Reply:
It seems it's out of our control, they are annotations inside `javax.validation`, @lukaszlenart . It seems we have to use `Default.class` when we want to override default `groups` but we also want to have Default in our basket.

Reply:
I was referring to the the `ValidataGroup` annotation, to put `Default.class` as a default value.

Reply:
if i add the Default.class to the default value in the Annotation (ValidationGroup), it becomes a depencies to the javax.validation package. Is this a good practise?